uint8 version
uint8 layer
uint16 position
uint32 keys
---
uint8 version
uint8 layer
uint16 position
uint8[] cfgData